blog/config/nav.yml

```
@@ -43,6 +43,7 @@ nav:
           - releases/announcing-knative-v0-3-release.md
           - releases/announcing-knative-v0-2-release.md
       - Articles:
+          - articles/singel-node-kafka-development.md      
```

```suggestion
          - articles/single-node-kafka-development.md      
```
could you rename the file?